quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -43,6 +43,21 @@ public final class PropertyMappers {
     public static String VALUE_MASK = "*******";
     private static MappersConfig MAPPERS;
     private static final Logger log = Logger.getLogger(PropertyMappers.class);
+    private final static List<PropertyMapperGrouping> GROUPINGS;
+    static {
+        GROUPINGS = List.of(new CachingPropertyMappers(), new DatabasePropertyMappers(),
```

Comment:
For the future, we could maybe use some Service Loader/SPI for these mappers as users could add their own mappers/options instead of having the loong SPI props for their extensions. 

@shawkins WDYT?

Reply:
> Allowing extensions to contribute property mappers via service loading would open up a significant amount of supported surface area. It would make more sense to refine the existing facilities for how extensions advertise option metadata.

+1